packages/augur-ui/src/modules/modal/components/modal-view.tsx

```diff
@@ -184,6 +187,8 @@ const ModalView = ({
     }
   };
 
+  const appStatus = useAppStatusStore();
```

Comment:
I would break this out as `const { modal } = useAppStatusStore();` since we will never or should never be passing a modal to modalView, so the modal should always come from the app status state.